README.md

```diff
@@ -12,31 +12,7 @@ When you run `tapioca sync` in a project, `tapioca` loads all the gems that are
 
 ## Manual gem requires
 
-For gems that have a normal default `require` and load all of their constants through such a require, everything works seamlessly. However, for gems that are marked as `require: false` in the Gemfile, or for gems that export optionally loaded types via different requires, where a single require does not load the whole gem code into memory, `tapioca` will not be able to load some of the types into memory and, thus, won't be able to generate complete RBIs for them. For this reason, we need to keep a small external file named `sorbet/tapioca/require.rb` that is executed after all the gems in the Gemfile have been required and before generation of gem RBIs have started. This file is responsible for adding the requires for additional files from gems, which are not covered by the default require.
```

Comment:
I think I would still like to keep this in the README for now. It is a unique feature of Tapioca and should be quite prominent in the README IMO. We can think about how we want to change the README afterwards, if you want, but let's keep this here as well, for now.

Reply:
Can do!